config/viper.go

```diff
@@ -82,3 +84,8 @@ func setWebsocketDefaults() {
 	viper.SetDefault(EnvWebsocketHistoryMax, 300)
 	viper.SetDefault(EnvWebsocketHistoryTtl, 10)
 }
+
+func setContactsDefaults() {
+	viper.SetDefault(EnvContactsPasscodePeriod, uint(3600)) //1h
+	viper.SetDefault(EnvContactsPasscodeDigits, uint(4))
```

Comment:
I'm almost certain that if we demonstrate this to someone from BA, there will be a question: *Is it possible to use two digits for TOTP?*

Reply:
So, what would you recommend?

Reply:
Let's use 2 digits - same pattern as in the mockups

Reply:
Changed 👍 
Also adjusted in frontend